core/src/modules/modelName.ts

```diff
@@ -0,0 +1,8 @@
+import { Model } from "sequelize-typescript";
+import { LoggedModel } from "../classes/loggedModel";
+
+export function modelName(instance: Model | typeof LoggedModel) {
+  let name = instance.constructor.name;
+  name = name[0].toLowerCase() + name.substr(1);
+  return name;
+}
```

Comment:
We end up using this a lot within the polymorphic helper (options, mappings, filters)... now with proper Types!